src/lib/svg_text_utils.js

```diff
@@ -173,70 +173,149 @@ function cleanEscapesForTex(s) {
         .replace(GT_MATCH, '\\gt ');
 }
 
+var inlineMath = [['$', '$'], ['\\(', '\\)']];
+
 function texToSVG(_texString, _config, _callback) {
+    var MathJaxVersion = parseInt(
+        (MathJax.version || '').split('.')[0]
+    ) || -1;
+
+    if(!MathJaxVersion) {
+        Lib.warn('No MathJax version:', MathJax.version);
+    }
+
     var originalRenderer,
         originalConfig,
         originalProcessSectionDelay,
         tmpDiv;
 
-    MathJax.Hub.Queue(
-    function() {
-        originalConfig = Lib.extendDeepAll({}, MathJax.Hub.config);
+    var setConfig = function() {
+        if(MathJaxVersion < 3) {
```

Comment:
Some of these functions don't share anything internal between 2 and 3, and they get invoked separately for the two cases - might be cleaner and more readable to make separate functions `setConfig2` and `setConfig3`.

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/6073/commits/7677258b8ef5a1579167cbfc84bcbd7ade94311e.